### PR TITLE
[benchmarks] Update dependencies

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

**Comment by pvdz** [comment]
Thanks!
I think I'd prefer the benchmark plugin to stay at `*` since we always want to make sure we use the latest version of that plugin when we run our benchmarks. Could you please revert those bits? Rest lgtm.

File: benchmarks/markdown_id/package.json [hunk 1]
```diff
@@ -35,8 +35,7 @@
   },
   "devDependencies": {
     "faker": "^4.1.0",
-    "gatsby-plugin-benchmark-reporting": "*",
-    "prettier": "^1.19.1"
+    "gatsby-plugin-benchmark-reporting": "^0.0.1"
```
Comment (pvdz) [inline benchmarks/markdown_id/package.json]: Please revert this line

File: benchmarks/markdown_slug/package.json [hunk 2]
```diff
@@ -35,8 +35,7 @@
   },
   "devDependencies": {
     "faker": "^4.1.0",
-    "gatsby-plugin-benchmark-reporting": "*",
-    "prettier": "^1.19.1"
+    "gatsby-plugin-benchmark-reporting": "^0.0.1"
```
Comment (pvdz) [inline benchmarks/markdown_slug/package.json]: Please revert this line

File: benchmarks/markdown_table/package.json [hunk 3]
```diff
@@ -9,14 +9,14 @@
...
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0"
   },
   "devDependencies": {
     "faker": "^4.1.0",
-    "gatsby-plugin-benchmark-reporting": "*",
-    "gray-matter": "^4.0.1"
+    "gatsby-plugin-benchmark-reporting": "^0.0.1",
```
Comment (pvdz) [inline benchmarks/markdown_table/package.json]: Please revert this line

File: benchmarks/query/package.json [hunk 4]
```diff
@@ -10,12 +10,12 @@
...
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "gatsby": "^2.19.5",
+    "lodash": "^4.17.15",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0"
   },
   "devDependencies": {
-    "gatsby-plugin-benchmark-reporting": "*"
+    "gatsby-plugin-benchmark-reporting": "^0.0.1"
```
Comment (pvdz) [inline benchmarks/query/package.json]: Please revert this line

File: benchmarks/plugin-manifest/package.json [hunk 5]
```diff
@@ -12,10 +12,10 @@
...
   "license": "MIT",
   "dependencies": {
-    "fs-extra": "^7.0.1",
-    "gatsby-plugin-manifest": "^2.2.37"
+    "fs-extra": "^8.1.0",
+    "gatsby-plugin-manifest": "^2.2.38"
   },
   "devDependencies": {
-    "gatsby-plugin-benchmark-reporting": "*"
+    "gatsby-plugin-benchmark-reporting": "^0.0.1"
```
Comment (pvdz) [inline benchmarks/plugin-manifest/package.json]: Please revert this line

File: benchmarks/image-processing/package.json [hunk 6]
```diff
@@ -5,26 +5,24 @@
...
-    "react-helmet": "^5.2.1",
-    "unsplash-js": "^5.0.0"
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-helmet": "^5.2.1"
   },
   "devDependencies": {
-    "gatsby-plugin-benchmark-reporting": "*",
-    "prettier": "^1.18.2"
+    "gatsby-plugin-benchmark-reporting": "^0.0.1"
```
Comment (pvdz) [inline benchmarks/image-processing/package.json]: Please revert this line

File: benchmarks/create-pages/package.json [hunk 7]
```diff
@@ -9,11 +9,11 @@
...
-    "gatsby": "^2.18.25",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "gatsby": "^2.19.5",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0"
   },
   "devDependencies": {
-    "gatsby-plugin-benchmark-reporting": "*"
+    "gatsby-plugin-benchmark-reporting": "^0.0.1"
```
Comment (pvdz) [inline benchmarks/create-pages/package.json]: Please revert this line

**Comment by MichaelDeBoey** [comment]
@pvdz I've deleted the `*`, so we're enforced to have a look at the benchmarks when we have breaking changes/compatibility.

**Comment by pvdz** [comment]
I appreciate that, however the benchmark plugin, by design, should never ship anything that prevents the benchmarks/gatsby from running. It'd be the opposite that I'd need to know about (Gatsby introducing a breaking change that breaks the plugin), but that wouldn't be caught by this change.

I'll discuss this internally to double check.

**Comment by pvdz** [comment]
We discussed it and would like to keep the star. It makes sense in this context. The benchmark plugin is my project and I expect it to change quite a bit in the next few months. There just isn't a sane point in having to churn all benchmark package jsons every time that happens. And once that does matter we can still reconsider that later.

So please revert those stars. The rest is good and can be merged. Thank you!

**Comment by MichaelDeBoey** [comment]
@pvdz So only the star of the benchmark plugin? 🤔

**Comment by pvdz** [comment]
Yes, we'd like to keep `"gatsby-plugin-benchmark-reporting": "*",` in all packages. The rest was good. Thanks!

**Comment by MichaelDeBoey** [comment]
@pvdz Updated the benchmark plugin dependency 🙂

**Comment by MichaelDeBoey** [comment]
With pleasure 🙂

**Pull request merged**